pkg/apis/extensions/v1alpha1/types_containerruntime.go

```diff
@@ -66,6 +66,10 @@ type ContainerRuntimeList struct {
 type ContainerRuntimeSpec struct {
 	// BinaryPath is the Worker's machine path where container runtime extensions should copy the binaries to.
 	BinaryPath string `json:"binaryPath"`
+	// WorkerGroup specifies the worker group the ContainerRuntime should be available for.
+	WorkerGroup string `json:"workerGroup"`
+	// WorkerGroupSelector is the label selector the extension uses to match the nodes belonging to the worker group.
+	WorkerGroupSelector metav1.LabelSelector `json:"workerGroupSelector"`
```

Comment:
Can we combine them and call them "worker pool"?

```yaml
workerPool:
  name: <name>
  selector:
    matchLabels:
      ...
```

Reply:
Adjusted it.